stellarphot/visualization/comparison_functions.py

```diff
@@ -707,11 +715,23 @@ def save_tess_files(self, button=None):
         """
         if self._field_name.value:
             self.tess_field_view()
-            self.iw.save(self._field_name.value, overwrite=True)
+            # Remove output file if it exists
+            if Path(self._field_name.value).exists() and self.overwrite_outputs:
+                Path(self._field_name.value).unlink()
+            try:
+                self.iw.save(self._field_name.value)
+            except:
```

Comment:
Usually it is better to not use a bare `except`, I think, in case you accidentally capture an error message that is really a bug. I think we could get away with no try/except here, anticipating that if an error is raised it will probably be for something real, and should be reported to the user with the exception details/traceback.

Reply:
Oh crap, I forgot to capture the specific error.  I ended up wrapping all the file creation calls that could fail due to attempting to overwrite a file without permission with a try..except OSError.  That should be specific enough.  I notice the errors still show up on the Jupyterlab log page, so may not really be much easier for users.